pyproject.toml

```diff
@@ -77,6 +77,7 @@ dev = [
     "types-jsonschema",
     "types-setuptools",
     "geopandas",
+    "polars",
```

Comment:
The original goal of the `all` dependeny group was to provide an easy way for users to set up all dependencies they are likely to use together with Altair. See https://github.com/vega/altair/pull/3354. I'm not sure if we want to include pandas/numpy/polars here or instead only in `dev` for the tests. To me, there is a good case for pandas and numpy as almost all examples in the Altair docs use Pandas. We want one install command which can be used in workshops which gets everything up and running. As there is no content so far for Polars, maybe we'd just move that one to `dev` for now?

If Altair and Polars get more integrated in the future, we should also have comprehensive examples with Polars in the docs and could reconsider moving it to `all`.

Any thoughts on this? Pinging @jonmmease @mattijn @joelostblom as all of you have been involved in this discussion at one point.

Reply:
It's probably good to mention `pandas` explicitly in the `dev` group too. 

I don't think it is necessary to have a dataframe library directly included within `all`. Meanwhile, pandas is still a dependency of `vega-datasets` and `vegafusion`, so it still will be installed when installing the full `altair-all` (just like `pyarrow` btw).

Reply:
@mattijn @binste 

Are you aware of the test dependencies feature in `hatch`?
https://hatch.pypa.io/latest/config/internal/testing/#dependencies

Reply:
Sounds interesting! Especially in combination with https://hatch.pypa.io/latest/how-to/integrate/vscode/, so it is possible to debug/run single tests.